{{cookiecutter.project_slug}}/backend/app/api/api_v1/routers/tests/test_auth.py

```diff
@@ -14,6 +15,25 @@ def verify_password_mock(first: str, second: str):
     assert response.status_code == 200
 
 
+def test_signup(client):
+    response = client.post(
```

Comment:
This should also use a monkeypatch, the example I have above should work. In this case it's a different function though (get_password_hash)